x-pack/plugins/observability_solution/observability_ai_assistant/public/components/chat/chat_flyout.tsx

```diff
@@ -213,10 +193,19 @@ export function ChatFlyout({
 
             {conversationsExpanded ? (
               <ConversationList
-                selected={conversationId ?? ''}
-                onClickDeleteConversation={handleClickDeleteConversation}
-                onClickChat={handleClickChat}
-                onClickNewChat={handleClickNewChat}
+                conversations={conversationList.conversations}
+                isLoading={conversationList.isLoading}
+                selectedConversationId={conversationId}
+                onConversationDeleteClick={(deletedConversationId) => {
+                  conversationList.deleteConversation(deletedConversationId).then(() => {
+                    if (deletedConversationId === conversationId) {
+                      setConversationId(undefined);
+                    }
+                  });
+                }}
```

Comment:
nit: can we use async/await for event handlers?
```suggestion
                onConversationDeleteClick={async (deletedConversationId) => {
                  await conversationList.deleteConversation(deletedConversationId);
                  if (deletedConversationId === conversationId) {
                    setConversationId(undefined);
                  }
                }}
```

Reply:
The handlers are typed as ( ) => void, I prefer to be explicit here.